src/components/Main/Articles/Articles.jsx

```diff
@@ -1,26 +1,32 @@
-import React, { useState, useEffect } from 'react';
+
+import  { useState, useEffect } from 'react';
 import axios from 'axios';
+import ArticleCard from './ArticleCard';
+import './Article.css';
 
 const Articles = () => {
   const [articles, setArticles] = useState([]);
 
   useEffect(() => {
-
     axios.get("https://nc-news-vvdv.onrender.com/api/articles")
       .then(response => setArticles(response.data.articles))
       .catch(error => console.error('Error fetching articles:', error));
   }, []);
-
+ if (articles.length === 0) return <p>Loading...</p>;
```

Comment:
lovely! If you want you can bring this out into a Loading component which will make it reusable